constraints/utils.js

```
@@ -11,7 +11,15 @@ export const applyConstraintToRecord = (c, r, f, d) => {
   const constraint = f.constraints?.find((fc) => fc.validator === c.validator)
   const { config = {} } = constraint || {}
   try {
-    eval(c.function)(r.get(f.key), f.key, { config, record: r, deps: d })
+    const constraintFn = c.function.startsWith('function')
+      ? c.function.includes('function constraint')
+        ? eval(
```

Biome

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)